Zend/zend_fibers.c

```diff
@@ -66,15 +66,13 @@ static zend_object_handlers zend_fiber_handlers;
 
 static zend_function zend_fiber_function = { ZEND_INTERNAL_FUNCTION };
 
-typedef void *fcontext_t;
+typedef struct _zend_fiber_transfer {
```

Comment:
Do you really want to rename these? I think we will benefit more from using the exact same naming as in [boost.context](https://github.com/boostorg/context/blob/d9dc03998b62b0ffcf8163b86b987fcbdd1a79c9/include/boost/context/detail/fcontext.hpp#L25). While the names they have chosen are not great I think it makes sense to be consistent in case we ever need to lookup things. Plus these types are not exported and cannot be used anywhere outside of `zend_fibers.c`, using a `zend_` prefix makes it look like they are coming from some header file...

Reply:
I have one more PR coming up that uses a type called `zend_fiber_transfer` and makes it part of the fiber API, I would need to be very creative and come up with a better name if you change the name here.

Reply:
It doesn't matter much what this name is as it's internal. Someone in R11 chat said using `transfer_t` could conflict, but maybe they were thinking it was exported. Feel free to name it whatever you want, maybe `zend_fiber_asm_transfer`. We definitely can't use something as general as `fcontext_t` in the header. Perhaps a comment noting what they are equivalent to in boost.context?

Reply:
I renamed primarily so `handle` in the context was of type `void*` where you might wonder what it's for… but a comment maybe would better too since no one should modify that pointer anyway. Reverting this commit is fine if you want, as you said, they aren't exported anyway.